packages/alfa-table/src/table.ts

```diff
@@ -331,7 +332,9 @@ export namespace Table {
             this._slots[x] = [];
           }
           for (let y = cell.anchor.y; y < cell.anchor.y + cell.height; y++) {
-            this._slots[x][y] = Some.of(cell);
+            this._slots[x][y] = (this._slots[x][y] ?? Set.empty())
+              .reject((item) => item.equals(cell))
```

Comment:
This method will cause the entire set to be reconstructed.

Reply:
Which is why I ended up mutating the cell builder instead…
(also the set is supposed to have 0 or 1 element, unless there is a table model error)

Reply:
Yeah, that does seem to confirm my suspicions in https://github.com/Siteimprove/alfa/pull/469#discussion_r509235445. I need to have a much deeper look at this package before deciding on an approach here 🤔 Given that this deals with an edge case that we've at least diagnosed as relating to invalid table markup, I'll probably not get around to it until after the stuff I'm currently prioritising is out of the way.